include/yaml-cpp/exceptions.h

```diff
@@ -120,6 +120,9 @@ class Exception : public std::runtime_error {
  private:
   static const std::string build_what(const Mark& mark,
                                       const std::string& msg) {
+    if (mark.is_null())
```

Comment:
Please add curly braces for this "if" statement, even if it's a one-line fix. I've been doing that with new code - I wish there was a way of telling clang-format to do this automatically. I may go through and update all of the one-line control flow statements at some point, but for now, I prefer new code to be in the proper format.